packages/web-components/src/text-input/README.md

```diff
@@ -0,0 +1,134 @@
+# TextInput
+
+> An implementation of a [text input](https://w3c.github.io/html-reference/input.text.html) as a form-connected web-component.
+
+<br />
+
+## **Design Spec**
+
+[Link to Text Input Design Spec in Figma](https://www.figma.com/file/TvHmWjZYxwtI9Tz6v5BT7E/Input?node-id=2366-657&t=UNSOfCD3St9ffppx-0)
+
+<br />
+
+## **Engineering Spec**
+
+Fluent WC3 Text Input extends from the [FAST Text Field](https://explore.fast.design/components/fast-text-field) and is intended to be as close to the Fluent UI React 9 Input implementation as possible. However, due to the nature of web components there will not be 100% parity between the two.
+
+<br />
+
+## Class: `TextInput`
+
+<br />
+
+### **Component Name**
+
+`<fluent-text-input>`
+
+<br />
+
+### **Variables**
+
+| Name                  | Description                          | Type                                                                                                             |
+| --------------------- | ------------------------------------ | ---------------------------------------------------------------------------------------------------------------- |
+| `TextInputSize`       | Size variations for text input       | `{ small: "small", medium: "medium", large: "large" }`                                                           |
+| `TextInputAppearance` | Appearance variations for text input | `{ outline: "outline", underline: "underline", filledLighter: "filled-lighter", filledDarker: "filled-darker" }` |
+| `TextInputType`       | Text input types                     | `{ email: "email", password: "password", tel: "tel", text: "text", url: "url" }`                                 |
+| `TextInputLayout`     | Layout variations for text input     | `{ block: "block", inline: "inline"`                                                                             |
```

Comment:
I see the layout in the design spec but not in FUIR9 implementation, not sure it is needed. Do we want to add it now or omit it and eventually add later?

Reply:
In the case of WC's, I'm not sure this makes a ton of sense in general personally assuming that it changes the "host" layout. We have the concept of `block` layout, etc on a few components it seems which changes the display property of them. It would be great to consider if it's worth refactoring so that CSS could be used simply and efficiently rather than needing to setup an attribute, etc. 

Regardless of the above convo, I'm for *not* including this if it's not part of the react v9 implementation at this time.

Reply:
I've removed `layout` from the spec as well as in the component PR..

[TextInput PR](https://github.com/microsoft/fluentui/pull/27348)